Assets/Tests/InputSystem/Plugins/EnhancedTouchTests.cs

```diff
@@ -798,6 +798,41 @@ public void EnhancedTouch_CanReceiveCallbacksOnFingerActivity()
         }));
     }
 
+    [Test]
+    [Category("EnhancedTouch")]
+    public void EnhancedTouch_TouchWorksAfterDisableEnable()
```

Comment:
Preferably something like `EnhancedTouch_CanBeDisabledAndReenabled`.

Also, for tests coming out of tickets, preferably put a comment with a link to the ticket in the code.

Reply:
I changed the name, but I think new name doesn't represent what the test is doing, the test only checks that touch phases specifically work, not whole enhanced touch module.